<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sonar_Widget</class>
 <widget class="QWidget" name="Sonar_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="portNum">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>9007</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>sonar 연결</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverIP">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.101.10</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>UltraSonic:  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="SonarData">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SonarData</string>
   </property>
  </widget>
  <widget class="QWidget" name="sensor" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>661</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QLabel" name="strMsg">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SonarData</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
